tests/foreman/cli/test_contenthost.py

```diff
@@ -567,115 +559,38 @@ def setUp(self):
 
```

Comment:
@abalakh and @sthirugn I don't see any issue on using the setUpClass and setUp methods on the same TestCase class if they are used the way they should be used.

There is just one concern here, if something fail on setUpClass then the entire class will not run while if setUp fail just that specific test will not run.

An approach to workaround this would be creating class attributes, for example:

```python
class MyTestCase(UITestCase):
    org = None
    env = None
    content_view = None
    activation_key = None

    def setUp(self):
        if MyTestCase.org is None:
             MyTestCase.org = create_org() # if fail then MyTestCase.org should be set to None
        # ...
```